packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -968,6 +968,10 @@ export default class Meeting extends StatelessWebexPlugin {
         EVENT_TRIGGERS.MEETING_INFO_AVAILABLE
       );
 
+      Metrics.sendBehavioralMetric(
```

Comment:
Can you run this with @Neeraj-swarnkar 

i dont think its the right place

Reply:
As per our discussion Metrics are added in `meetingInfoProvider.fetchMeetingInfo` which we are mocking in the unit test, as a result we will need to remove assertion for Metrics calls.